vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantToVariantContextConverter.java

```diff
@@ -86,57 +96,61 @@ public VariantContext transform(Variant variant) {
         return variantContext;
     }
 
-    private String getAnnotationAttributes(Variant variant) {
-        List<ConsequenceType> consequenceTypes = variant.getAnnotation().getConsequenceTypes();
-        String allele = variant.getAnnotation().getAlternativeAllele();
+    private String getAnnotationAttributes(VariantWithSamplesAndAnnotation variant) {
+        Annotation annotation = variant.getAnnotation();
+        Set<ConsequenceType> consequenceTypes = null;
+        if (annotation != null) {
+            consequenceTypes = annotation.getConsequenceTypes();
+        }
         String csq = null;
         if (consequenceTypes != null) {
-            List<String> consequences = new ArrayList<>();
-            for (ConsequenceType consequenceType : consequenceTypes) {
-                List<ConsequenceType.ConsequenceTypeEntry> soTermList = consequenceType.getSoTerms();
-                List<String> soNameList = soTermList.stream().map(ConsequenceType.ConsequenceTypeEntry::getSoName)
-                        .collect(Collectors.toList());
-                String soNames = String.join("&", soNameList);
-                String symbol = consequenceType.getGeneName();
-                String gene = consequenceType.getEnsemblGeneId();
-                String feature = consequenceType.getEnsemblTranscriptId();
-                String bioType = consequenceType.getBiotype();
-                Integer cDnaPosition = consequenceType.getcDnaPosition();
-                Integer cdsPosition = consequenceType.getCdsPosition();
-
-                StringBuilder csqSb = new StringBuilder();
-                csqSb.append(allele != null ? allele : "").append("|")
-                        .append(soNames != null ? soNames : "").append("|")
-                        .append(symbol != null ? symbol : "").append("|")
-                        .append(gene != null ? gene : "").append("|")
-                        .append(feature != null ? feature : "").append("|")
-                        .append(bioType != null ? bioType : "").append("|")
-                        .append(cDnaPosition != null ? cDnaPosition : "").append("|")
-                        .append(cdsPosition != null ? cdsPosition : "");
-                consequences.add(csqSb.toString());
-            }
-            csq = String.join(",", consequences);
-
+            String allele = variant.getAlternate();
```

Comment:
The 'allele' variant can be inlined: 
` ... transformConsequencetypeToCsqTag(variant.getAlternate(),consequenceType)....`
A separate variable was needed with the previous method structure but not anymore.